app/Http/Controllers/GistsController.php

```diff
@@ -93,4 +94,13 @@ protected function unstar(GistClient $client, $gistId)
             ], 500);
         }
     }
+
+    public function starCount(GistClient $client, $gistId)
```

Comment:
Now I'm wondering... do we have our own GitHub key we can use to look up the star counts, so even guests can see them?

Reply:
I was mulling over something like this, the only downside would be potentially hitting the built in limits. I think its 5000 "points" per hour for graphql API. I know that 1 point doesn't equal 1 call so I'm not sure how many points our little query uses but depending on how many people use gistlog its a limit that might be hit